packages/react-make-styles/src/makeStyles.ts

```diff
@@ -23,14 +22,15 @@ export function makeStyles<Slots extends string>(stylesBySlots: Record<Slots, Ma
     return () => ({} as Record<Slots, string>);
   }
 
-  return function useClasses(): Record<Slots, string> {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  return function useClasses(unstable_themeOverride?: Theme): Record<Slots, string> {
```

Comment:
This parameter should never be used except in Theme Provider. Therefore I don't like it being added.
I would rather create a second version of `makeStyles` (`makeStylesWithCustomTheme`) to be used only in the Theme Provider (and ideally call `makeStylesWithCustomTheme` from the regular `makeStyles`.